src/viam/utils.py

```diff
@@ -351,3 +265,8 @@ def from_dm_from_extra(extra: Optional[Dict[str, Any]]) -> bool:
         return False
 
     return bool(extra["fromDataManagement"])
+
+
+_numpy_import_error_string = """MLModel support in Viam Python SDK requires the installation of optional
```

Comment:
This would probably live in the `services/mlmodel/__init__.py` file, and maybe with the name `_requirements_not_met_error_string` or something similar (basically just not specifying `numpy` by name, in case of future dependency changes)